itest/lnd_config.go

```diff
@@ -0,0 +1,28 @@
+//go:build integration
+
+package itest
+
+import (
+	"os/exec"
+
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/stretchr/testify/require"
+)
+
+// testDebuglevelShow tests that "lnd --debuglevel=show" command works and
+// prints the list of supported subsystems.
+func testDebuglevelShow(ht *lntest.HarnessTest) {
```

Comment:
def need tests like this! Tho I feel like we could put it in `config.go` as it's not testing any integrating behavior...future PRs I guess.

Reply:
I renamed this file to `itest/config.go`